blog-service/2024-03-18-dashboards.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Dashboard Enhancements - Crosshair and Time Syncing (Alerts)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - dashboards
+  - release notes
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're happy to announce two new dashboarding features:
+
+### Synced Crosshairs
+When hovering over a time series panel Sumo Logic now displays a related crosshair in all other time series panels within the dashboard. With the synced crosshairs you can quickly see how the data in one panel may relate to what you see within other panels.
+ 
+
+
+### Send Panel Zoom to Dashboard
+When zooming into a panel, Sumo Logic now provides an option to temporarily set the zoomed timerange as the dashboard time range. This allows you to quickly zoom all other panels of the dashboard to the same time range. 
```

Comment:
Should it be _timerange_ or _time range_?

```suggestion
When zooming into a panel, Sumo Logic now provides an option to temporarily set the zoomed timerange as the dashboard time range. This allows you to quickly zoom all other panels of the dashboard to the same time range. 
```

Reply:
Content updated.

Reply:
thank you!